quantum/audio/audio.h

```diff
@@ -87,9 +87,10 @@ void play_notes(float (*np)[][2], uint16_t n_count, bool n_repeat, float n_rest)
 
 bool is_playing_notes(void);
 
-void play_goodbye_tone(void);
-void play_startup_tone(void);
+void startup_song_user(void);
```

Comment:
Do we really need a dedicated song function with the `matrix_init_user`? We could call it `keyboard_init_user` and move it outside of `matrix.c` and into `/tmk_core/common/keyboard.c`. Maybe `quantum_init`, `user_init`? `keyboard_init` is already taken..